cmd/context/utils_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"testing"
 
 	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/model/contextresource"
```

Comment:
Does this need to be a sub-package of model? I like the idea of creating a better package name with a purpose but adding it as a sub-package doesn't really help eliminate the `model` package which is the direction I'd like to go.